### ISSUE TITLE
[⚡ Feature]:  Allow npm install --force

### ISSUE THREAD

**Issue by coratype** [cloudflare/next-on-pages#694]
### Description

Dependency conflicts break the npx @cloudflare/next-on-pages@1 command, in some cases it would be nice to be able to use --force despite the conflict for local testing

### Would you like to help?

- [ ] Would you like to help implement this feature?

**Comment by santhonydo** [comment]
yup im having the same problem with svelte 5 (currently in release candidate status)

**Comment by dario-piotrowicz** [comment]
Hello 🙂 Thank you for the issue and apologies for the late reply.

Earlier this year we released our [Cloudflare adapter for OpenNext](https://opennext.js.org/cloudflare), and we have been continuously investing effort in its improvement since then. Existing deployments can continue to use the next-on-pages tooling, but the Cloudflare adapter for OpenNext is currently stable and recommended for deployment of Next.js applications to Cloudflare.

As such, we will no longer be maintaining next-on-pages. We recently deprecated the package on npm and we plan to archive this repo on Monday Sep 29, 2025. In preparation for archival, we are closing all open issues and PRs.

If you have concerns, please feel free to reply here before the repo is archived (after that, it will no longer be possible to comment) or to reach out to our team on [Discord](https://discord.com/invite/cloudflaredev).

Thanks so much for using and contributing to next-on-pages 🧡
